drivers/storage/portworx/util/util.go

```diff
@@ -1570,3 +1576,49 @@ func GetTLSCipherSuites(cluster *corev1.StorageCluster) (string, error) {
 	}
 	return strings.Join(outList, ","), nil
 }
+func GetKvdbMap(k8sClient client.Client,
+	cluster *corev1.StorageCluster,
+) map[string]*kvdb_api.BootstrapEntry {
+	// If cluster is running internal kvdb, get current bootstrap nodes
+	kvdbNodeMap := make(map[string]*kvdb_api.BootstrapEntry)
+	if cluster.Spec.Kvdb != nil && cluster.Spec.Kvdb.Internal {
+		clusterID := GetClusterID(cluster)
+		strippedClusterName := strings.ToLower(ConfigMapNameRegex.ReplaceAllString(clusterID, ""))
+		cmName := fmt.Sprintf("%s%s", InternalEtcdConfigMapPrefix, strippedClusterName)
+
+		cm := &v1.ConfigMap{}
+		err := k8sClient.Get(context.TODO(), types.NamespacedName{
+			Name:      cmName,
+			Namespace: "kube-system",
```

Comment:
Let's redeclare the constant `BootstrapCloudDriveNamespace` in this file and change the existing references to it.

Reply:
Updated